pilot/pkg/networking/core/cluster_waypoint.go

```diff
@@ -246,6 +283,12 @@ func (cb *ClusterBuilder) buildWaypointInboundVIP(proxy *model.Proxy, svcs map[h
 			if port.Protocol == protocol.UDP {
 				continue
 			}
+			if isEastWestGateway(proxy) {
+				// East-west gateways don't respect DestinationRule, so don't read it here
```

Comment:
This comment seems a bit misleading since in buildWaypointInboundVIPCluster we do apply DR... maybe we just want to skip the subsets?

Reply:
Ah nvm its even weirder -- we pass policy `nil` but then call things like applyOutlierDetection, applyConnectionPool, etc. This may end up being correct (not sure if those apply some defaults, etc) but its confusing at least.

Reply:
Yeah the idea was to keep the defaults and not have to update them in two places moving forward